fully working swagger fetching

auppunda opened the pull request.

pacogomez commented:
code fails with
`container_service_extension/processor.py”, line 123, in get_swagger_json_file
    raise Exception(“Swagger file not found”)
Exception: Swagger file not found`

auppunda commented:
this should work but don't merge yet, I am still testing it.

auppunda commented:
packages swagger file with package_data in setup.py, changed get json file and yaml file methods to reflect the change in the way it was packaged, and removed swagger as a data file
